pkg/traces/exporter/exporter.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/kolide/launcher/pkg/agent/types"
 	"github.com/kolide/launcher/pkg/backoff"
 	"github.com/kolide/launcher/pkg/osquery"
+	osquerygotraces "github.com/osquery/osquery-go/traces"
```

Comment:
Seeing this, makes me think we should put `SetTracerProvider` to being in the toplevel `osquery-go` library. and leave `traces` as effectively internal. But :shrug: maybe not important

Reply:
I don't feel strongly about it either way

Reply:
I don't think it's worth fixing _now_, but it feels wrong. `traces` should really be an internal package, not part of the exposed API surface. But :shrug: we're most of the users, and I don't feel like we need to maintain some high standard of API consistency, so not worth it.